src/tribler/core/utilities/tracker_utils.py

```diff
@@ -116,11 +121,14 @@ def parse_tracker_url(tracker_url):
     if tracker_url.lower().startswith('http://') and ':80/' in tracker_url:
         tracker_url = tracker_url.replace(':80/', '/', 1)
 
+    if tracker_url.lower().startswith('https://') and ':443/' in tracker_url:
+        tracker_url = tracker_url.replace(':443/', '/', 1)
```

Comment:
The numbers `443` and `80` are hardcoded here. Also, there are duplications in strings (`':80/'` and `':443/'`)

Consider replacing them with something like:
```suggestion
    url = tracker_url.lower()
    http_port = f':{HTTP_PORT}/'
    https_port = f':{HTTPS_PORT}/'
    if url.startswith(f'http://') and http_port in url:
        tracker_url = tracker_url.replace(http_port, '/', 1)

    if url.startswith('https://') and https_port in url:
        tracker_url = tracker_url.replace(https_port, '/', 1)
```

Reply:
Addressed